### PR TITLE
add additional subject alt name fields

### PR DESCRIPTION
**Pull request by zzma**
@zakird - additional SAN field schema

### CODE REVIEW

File: ztag/schema.py [hunk 1]
```diff
@@ -150,6 +150,17 @@
...
+            "edi_party_names":ListOf(SubRecord({
+                "name_assigner":String(),
+                "party_name":String(),
+            })),
+            "other_names":ListOf(SubRecord({
+                "id":String(),
+                "value":String(),
+            })),
+            "registered_ids":ListOf(String()),
+            "uniform_resource_identifiers":ListOf(String()),
```
Comment (zakird) [inline ztag/schema.py]: This should probably be AnalyzedString(es_include_raw=True)

File: ztag/schema.py [hunk 2]
```diff
@@ -150,6 +150,17 @@
             "dns_names":ListOf(AnalyzedString()),
             "email_addresses":ListOf(String()),
             "ip_addresses":ListOf(String()),
+            "directory_names":ListOf(zgrab_subj_issuer),
+            "edi_party_names":ListOf(SubRecord({
```
Comment (zakird) [inline ztag/schema.py]: Can you give an example for one of these?

File: ztag/schema.py [hunk 3]
```diff
@@ -150,6 +150,17 @@
...
+                "name_assigner":String(),
+                "party_name":String(),
+            })),
+            "other_names":ListOf(SubRecord({
+                "id":String(),
+                "value":String(),
```
Comment (zakird) [inline ztag/schema.py]: Do you have an example?

File: ztag/schema.py [hunk 4]
```diff
@@ -150,6 +150,17 @@
...
+            })),
+            "registered_ids":ListOf(String()),
```
Comment (zakird) [inline ztag/schema.py]: Can you give an example?

File: ztag/schema.py (hunk 2 above)
Comment (zzma) [inline ztag/schema.py]:
```
"edi_party_names": [
      {
        "name_assigner": "Mother Nature",
        "party_name": "partyName"
      }
    ]
```

File: ztag/schema.py (hunk 3 above)
Comment (zzma) [inline ztag/schema.py]:
```
"other_names": [
      {
        "id": "1.3.6.1.4.1.11483.46",
        "value": "AgMIKgk="
      }
    ]
```

File: ztag/schema.py (hunk 1 above)
Comment (zzma) [inline ztag/schema.py]: will do

File: ztag/schema.py (hunk 2 above)
Comment (zakird) [inline ztag/schema.py]: Alright, these should both be AnalyzedString with included raw.

File: ztag/schema.py (hunk 3 above)
Comment (zakird) [inline ztag/schema.py]: value should be IndexedBinary not string

File: ztag/schema.py (hunk 4 above)
Comment (zzma) [inline ztag/schema.py]: this is an ASN1 OID, so it will look like `1.2.3.4.5`

**Comment by zzma** [comment]
@zakird - bump. Anything else?

**Comment by zakird** [comment]
LGTM

**Pull request merged**